helm/bold-common/README.md

```diff
@@ -163,6 +178,35 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ## Application Startup
 
-Configure the Bold BI application startup to use the application. Please refer the following link for more details on configuring the application startup.
+1. Configure the Bold BI application startup to use the application. Please refer the following link for more details on configuring the application startup.
     
-https://help.boldbi.com/embedded-bi/application-startup
+   https://help.boldbi.com/embedded-bi/application-startup
+
+2. Navigate to administration configuration file.
+   ```sh
+   <DNS>/ums/administration/config-editor
+   ```
+   ![Configuration-editor](/helm/docs/images/config-edit.png)
+
+3. In product.json add below lines for Bold Reports.
```

Comment:
3. In product.json add the lines below for Bold Reports.

Reply:
concern addressed